auction-app-backend/pom.xml

```diff
@@ -42,6 +42,11 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>3.0.0</version>
+		</dependency>
```

Comment:
Since you have already added dependency `io.springfox.springfox-boot-starter`, please check if we really need to add this one also. Swagger UI should be available on `host:port/swagger-ui/index.html` and `health` route should be shown.

Reply:
My mistake, we don't need the dependency. Is it OK to access Swagger UI with the prefix `/api/v1/`, or it needs to be strictly `host:port/swagger-ui/` ?